src/altair/networking.rs

```diff
@@ -0,0 +1,12 @@
+use super::SYNC_COMMITTEE_SUBNET_COUNT;
+use crate::phase0::ATTESTATION_SUBNET_COUNT;
+#[cfg(feature = "serde")]
```

Comment:
looks like this was accidentally copied, and will also fail the "no default features" build (cf. #191 )
```suggestion
```

Reply:
the same thing is below where I also left a suggestion

Reply:
I removed both lines, thanks!
However, related to #191, shouldn't I also add the `#[cfg_attr(feature = "serde", serde(with = "crate::serde::as_string"))]` in the Metadata struct? Or would that be managed on that PR. Let me know, thanks again.

Reply:
I'd say we resolve that other there, thanks!